cf/data/data.py

```diff
@@ -1415,7 +1415,7 @@ def _clear_after_dask_update(self, clear=_ALL):
             self._cfa_del_write()
 
     # REVIEW: getitem: `_set_dask`: new keyword 'asanyarray'
-    def _set_dask(self, dx, copy=False, clear=_ALL, asanyarray=False):
+    def _set_dask(self, dx, copy=False, clear=_ALL, __asanyarray__=False):
```

Comment:
I know we spoke about this today and agreed that a leading underscore to indicate internal-use intent for `asanyarray` is a good idea, but having both trailing and leading dunderscores for a name is meant to be reserved for special Python methods, so I don't think it is a good idea to have `__asanyarray__` for anything, unless you want to elevate `asanyarray` in these cases to the level of a 'magic' method? I personally think it is misleading so best keep it as `_anyanyarray` in these cases too.

Reply:
I agree - `__asanyarray__=` has gone!